src/main/java/org/gridsuite/directory/notification/server/DirectoryNotificationWebSocketHandler.java

```diff
@@ -154,18 +152,35 @@ private Flux<WebSocketMessage> heartbeatFlux(WebSocketSession webSocketSession)
                 .pingMessage(dbf -> dbf.wrap((webSocketSession.getId() + "-" + n).getBytes(StandardCharsets.UTF_8))));
     }
 
+    public Flux<WebSocketMessage> receive(WebSocketSession webSocketSession) {
+        return webSocketSession.receive()
+                .doOnNext(webSocketMessage -> {
+                    try {
+                        String wsPayload = webSocketMessage.getPayloadAsText();
```

Comment:
Deal wtih deletions (2 lists of filters) :
- filters to add
- filters to delete

Reply:
done